lib/saml.js

```diff
@@ -258,7 +258,7 @@ router.get('/login/ok', ensureAuthenticated, function(req, res) {
     first_name: lookupFromProfile('first_name', req.user),
     last_name: lookupFromProfile('surname', req.user),
     email: lookupFromProfile('email', req.user),
-    groups: lookupFromProfile('groups', req.user),
+    groups: lookupFromProfile('groups', req.user)? lookupFromProfile('groups', req.user) : [],
```

Comment:
`groups: lookupFromProfile('groups', req.user) || []` or even better, move this into `lookupFromProfile()` to DRY this up